apps/showcase/src/app/sdk-training/sdk-training.spec.ts

```diff
@@ -9,6 +9,15 @@ import {
 describe('SdkTrainingComponent', () => {
   let component: SdkTrainingComponent;
   let fixture: ComponentFixture<SdkTrainingComponent>;
+  global.fetch = jest.fn(() => Promise.resolve({
```

Comment:
Maybe cleaner to be set in the `beforeEach` to renew the mock before each test.
Or, if the single mock is done on purpose, maybe moving it outside of the `describe` as it is affecting globally.

(same comment for the other tests)